test/integration/targets/elb_target_info/playbooks/roles/elb_target_info/defaults/main.yml

```diff
@@ -1,7 +1,5 @@
 ---
```

Comment:
Thanks for fixing this role name. When this gets merged please also backport to 2.9.